raiden/transfer/mediated_transfer/events.py

```diff
@@ -127,7 +127,7 @@ class SendSecretRequest(SendMessageEvent):
     """
 
     payment_identifier: PaymentID
-    amount: PaymentWithFeeAmount
+    amount: PaymentAmount
```

Comment:
The SendSecretRequest can also include the remaining of fees that were not kept by mediators right? Still since they are not fees but just get added to the payment amount you change the type here?